etna/whatson/models.py

```diff
@@ -308,9 +317,18 @@ def get_context(self, request, *args, **kwargs):
         filter_form = EventFilterForm(request.GET)
 
         if filter_form.is_valid():
-            self.events = self.filter_form_data(filter_form.cleaned_data)
-
+            filtered_events = self.filter_form_data(filter_form.cleaned_data)
+            if self.featured_event:
+                context["events"] = filtered_events.exclude(id=self.featured_event_id)
+            else:
+                context["events"] = filtered_events
+        else:
+            context["events"] = self.events
         context["filter_form"] = filter_form
+        if self.featured_event:
+            context["show_fetured_event"] = self.should_show_featured_event(
+                filtered_events, self.featured_event
```

Comment:
Question: is this needed? We are filtering the featured event out of the listing, so shouldn't it always be shown in the featured card style?

Reply:
I think it is. In the template, we first render the featured event, then loop over the listing below to display the other events. We need to know whether the featured event should be filtered out or not.

Reply:
Got it, this was a little confusing, thanks!